Gesture_control.py

```diff
@@ -8,6 +8,34 @@
 import cv2
 import numpy as np
 
+
+def get_trackbars():
+    cv2.namedWindow('Trackbars')
+    cv2.resizeWindow('Trackbars', 640, 240)
+    cv2.createTrackbar('HueMin', 'Trackbars', 0, 179, empty)
```

Comment:
Define this empty function or else it would raise an error
You can make this function that simply passes.
Also, rather than printing the mask here return it so that we can use the result of this function.